app/(tabs)/notes.tsx

```diff
@@ -0,0 +1,86 @@
+import { Poppins_400Regular, useFonts } from '@expo-google-fonts/poppins';
+import { Redirect, useFocusEffect } from 'expo-router';
+import { useCallback, useState } from 'react';
+import { FlatList, SafeAreaView, StyleSheet } from 'react-native';
+import NoteItem from '../../components/NotesItem';
+import { colors } from '../../constants/colors';
+import type { Note } from '../../migrations/00003-createTableNotes';
+import type { NotesResponseBodyGet } from '../api/notes+api';
+import type { UserResponseBodyGet } from '../api/user+api';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: colors.background,
+    height: '100%',
+  },
+  text: {
+    color: colors.text,
+  },
+  list: {
+    marginTop: 30,
+  },
+});
+
+export default function App() {
+  const [notes, setNotes] = useState<Note[]>([]);
+  const [fontsLoaded] = useFonts({
+    Poppins_400Regular,
+  });
+  const [isStale, setIsStale] = useState(true);
```

Comment:
It seems like this `isStale` state variable is not necessary?

Reply:
Removed [5e63643](https://github.com/upleveled/expo-example-fall-2024-atvie/pull/1/commits/5e6364303ce95173da41d6c805c49c913293dead)

Reply:
@ProchaLu this removed the `isStale` for guests too.

But I guess that's wrong, because we still want to be able to change the state from the main guests page by clicking on one of the guests.

Reply:
Added `isStale` for guests
[3d96d7b](https://github.com/upleveled/expo-example-fall-2024-atvie/pull/1/commits/3d96d7b77873c33f0d3750f81683b5b69f8b4cf8)